src/utils/index.js

```diff
@@ -0,0 +1,11 @@
+export const isExpired = (date) => {
+  if (date === "never" || date === undefined) {
+    return false;
+  }
```

Comment:
It's a little clearer if you name the variable "expiryDate".

```suggestion
export const isExpired = (expiryDate) => {
  if (expiryDate === "never" || expiryDate === undefined) {
    return false;
  }
```